src/napari_matplotlib/scatter.py

```diff
@@ -222,3 +222,124 @@ def _on_update_layers(self) -> None:
         # reset the axis keys
         self._x_axis_key = None
         self._y_axis_key = None
+
+
+class FeaturesHistogramWidget(NapariMPLWidget):
+    n_layers_input = Interval(1, 1)
+    # All layers that have a .features attributes
+    input_layer_types = (
+        napari.layers.Labels,
+        napari.layers.Points,
+        napari.layers.Shapes,
+        napari.layers.Tracks,
+        napari.layers.Vectors,
+    )
```

Comment:
Could you put this definition in a new file (`features.py`?) called something like `FEATURES_LAYER_TYPES`, and then use that in both this new widget and the existing features scatter widget?